server/src/main/java/org/opensearch/index/remote/RemoteSegmentStats.java

```diff
@@ -173,7 +173,7 @@ public void add(RemoteSegmentStats existingStats) {
             this.downloadBytesFailed += existingStats.getDownloadBytesFailed();
             this.downloadBytesSucceeded += existingStats.getDownloadBytesSucceeded();
             this.maxRefreshTimeLag = Math.max(this.maxRefreshTimeLag, existingStats.getMaxRefreshTimeLag());
-            this.maxRefreshBytesLag = Math.max(this.maxRefreshBytesLag, existingStats.getMaxRefreshBytesLag());
+            this.totalRefreshBytesLag += existingStats.getTotalRefreshBytesLag();
```

Comment:
@sachinpkale  if we are able to plot the totalRefreshByteLag over time, we would be able to find the spikes and detect a faulty node. Tracking totalRefreshByteLag over time will help identify issues easily. Also, I am planning to use this totalRefreshByte lag in OSB which also provide the different percentile values.

Reply:
>  if we are able to plot the totalRefreshByteLag over time, we would be able to find the spikes and detect a faulty node.

It may not always be possible. Let's say a node has 10 shards each with 100KB lag at t0. At t1, 9 of the shards had lag 0 and 1 shard had 1MB. The value of `totalRefreshByteLag` would be same at t0 and t1.

Reply:
BTW, I am not suggesting to remove `totalRefreshByteLag`. We can have both.

Reply:
> > if we are able to plot the totalRefreshByteLag over time, we would be able to find the spikes and detect a faulty node.
> 
> It may not always be possible. Let's say a node has 10 shards each with 100KB lag at t0. At t1, 9 of the shards had lag 0 and 1 shard had 1MB. The value of `totalRefreshByteLag` would be same at t0 and t1.

True, however, we will be considering both the bytes and the time lag in determining how much the lag really is. Also, lag vs indexing volume might as well be crucial in determining how much percent of indexing data is yet to be uploaded to remote store. 

> BTW, I am not suggesting to remove totalRefreshByteLag. We can have both.

This should be fine as each metric will help uncover different section of issues.